### ISSUE TITLE
Using custom model with fewer that 1000 classes

### ISSUE THREAD

**Issue by dsanerph** [tensorflow/tensorflow#42948]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
Andoid studio 4.01 emulator will crash with error below if model has fewer classes than 1000

**Describe the problem**
I am trying to use a custom trained model that has fewer than 1000 classes - is there a way to change the number of classes? I see all the pretrained ones have 1000 classes

**Provide the exact sequence of commands / steps that you executed before running into the problem**

**Any other info / logs**
 java.lang.IllegalArgumentException: Label number 1005 mismatch the shape on axis 1 is the error I get because the output tensor of my model only includes 4 classes. Is this configurable? If not it would be a great feature to add.

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by Saduf2019** [comment]
@dsanerph 
Please provide with simple stand alone code such that we can replicate the issue faced, or if possible share a colab with error reported.

**Comment by ymodak** [comment]
You may use transfer learning approach to retrain the model as per your given classes.
Typically you will be using some pre-trained model architecture and set `include_top` as false.
This will not load the classification layer of the pre trained model.
This allows you add an output layer and specify units (as per your class requirements).
For more information see https://www.tensorflow.org/tutorials/images/transfer_learning
